decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -21,9 +21,12 @@ class Comment < ApplicationRecord
       validate :commentable_can_have_replies
       validates :depth, numericality: { greater_than_or_equal_to: 0 }
       validates :alignment, inclusion: { in: [0, 1, -1] }
+      validate :same_organization
 
       before_save :compute_depth
 
+      delegate :organization, to: :commentable
```

Comment:
Isn't this too arbitrary? Why whould you want to do this? You're pushing a dependency (`organization`) to the `commentable`, and I don't see much value in it. Shouldn't the comment itself contain the `organization` association instead?